src/containers/Storage/Storage.tsx

```diff
@@ -95,25 +89,15 @@ export const Storage = ({additionalNodesProps, tenant, nodeId}: StorageProps) =>
     const isNodePage = nodeId !== undefined;
     const storageType = isNodePage ? STORAGE_TYPES.groups : type;
 
-    const nodesRequestParams = useNodesRequestParams({
-        filter,
-        nodesUptimeFilter: uptimeFilter,
-        ...nodesSortParams,
-    });
-    const storageRequestParams = useStorageRequestParams({
-        filter,
-        ...groupsSortParams,
-    });
```

Comment:
Outdated hooks, in current implementation they always return `undefined`. They depended on old param `USE_BACKEND_PARAMS_FOR_TABLES_KEY` which was replaced with `USE_PAGINATED_TABLES_KEY`, so if `USE_PAGINATED_TABLES_KEY === false`, hooks return undefined, if `USE_PAGINATED_TABLES_KEY === true`, `PaginatedStorage` is used